src/bun.js/test/jest.zig

```diff
@@ -465,6 +735,58 @@ pub const Jest = struct {
     }
 };
 
+pub const ExpectAny = struct {
+    pub usingnamespace JSC.Codegen.JSExpectAny;
+
+    pub fn finalize(
+        this: *ExpectAny,
+    ) callconv(.C) void {
+        VirtualMachine.get().allocator.destroy(this);
+    }
+
+    pub fn call(globalObject: *JSC.JSGlobalObject, callFrame: *JSC.CallFrame) callconv(.C) JSC.JSValue {
+        const _arguments = callFrame.arguments(1);
+        const arguments: []const JSValue = _arguments.ptr[0.._arguments.len];
+
+        if (arguments.len == 0) {
+            globalObject.throw("any() expects to be passed a constructor function.", .{});
+            return .zero;
+        }
+
+        const constructor = arguments[0];
+        constructor.ensureStillAlive();
+        if (!constructor.isConstructor()) {
+            const fmt = "<d>expect.<r>any<d>(<r>constructor<d>)<r>\n\nExpected a constructor\n";
```

Comment:
ah maybe we should have `globalThis.throwPretty`

Reply:
good idea